src/index.js

```diff
@@ -22,10 +22,10 @@ const exec = thenify(nodeExec);
 
 export class NpmAutoInstallWebpackPlugin {
   constructor(options = {}) {
-    this.options = {
-      ...options,
-      autoInstall: options.autoInstall !== false,
-    };
+    this.options = Object.assign(options, {
+       install: true,
```

Comment:
This will make `options.install` always be `true`. Could you revert back to object spread syntax:

```js
{
   ...options,
   install: options.install !== false,
}
```